e2e/ansible/playbooks/feature/ha/ha.yml

```diff
@@ -105,7 +105,7 @@
            - name: Get storage ctrl pod name
              shell: >
                source ~/.profile;
-               kubectl get pods -n {{ operator_ns }} -l openebs.io/controller=cstor-controller | grep {{ namespace }}
+               kubectl get pods -n openebs -l openebs.io/target=cstor-target | grep {{ namespace }}
```

Comment:
We should continue to use operator_ns. It is possible that openebs components will be installed in a different namespace in future for cstor.